docs/conf.py

```diff
@@ -59,7 +59,6 @@
     "sphinx.ext.intersphinx",
     "sphinx.ext.linkcode",
     "sphinx.ext.mathjax",
-    "sphinx.ext.napoleon",
```

Comment:
ping @riedgar-ms re this

Reply:
Something I did _four years_ ago??? :-)

No objection from me so long as the docs still work.